### PR TITLE
Create mc2-histoqc-project

### PR DESCRIPTION
**Pull request by adamjtaylor**
This PR creates a Tower project for the MC2 Digital Pathology QC supplement

### CODE REVIEW

**Comment by adamjtaylor** [comment]
@thomasyu888 by default this got created in the `dev` branch. Is that correct?

**Comment by thomasyu888** [comment]
@adamjtaylor I will merge this into prod after it builds in dev.

File: config/projects-prod/mc2-histoqc-project.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+template:
+  path: tower-project.j2
+stack_name: mc2-mcmicro-project
```
Comment (thomasyu888) [inline config/projects-prod/mc2-histoqc-project.yaml]: @adamjtaylor Is the stack name wrong?

Comment (adamjtaylor) [reply config/projects-prod/mc2-histoqc-project.yaml]: Yes! Updated!

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**